src/java_bytecode/library/src/java/lang/Thread.java

```diff
@@ -0,0 +1,1987 @@
+   * value up or down as it sees fit (or to ignore it completely).
+   *
+   * <p>Specifying a value of zero for the {@code stackSize} parameter will
+   * cause this constructor to behave exactly like the
+   * {@code Thread(ThreadGroup, Runnable, String)} constructor.
+   *
+   * <p><i>Due to the platform-dependent nature of the behavior of this
+   * constructor, extreme care should be exercised in its use.
+   * The thread stack size necessary to perform a given computation will
+   * likely vary from one JRE implementation to another.  In light of this
+   * variation, careful tuning of the stack size parameter may be required,
+   * and the tuning may need to be repeated for each JRE implementation on
+   * which an application is to run.</i>
+   *
+   * <p>Implementation note: Java platform implementers are encouraged to
+   * document their implementation's behavior with respect to the
+   * {@code stackSize} parameter.
+   *
+   *
+   * @param  group
+   *         the thread group. If {@code null} and there is a security
+   *         manager, the group is determined by {@linkplain
+   *         SecurityManager#getThreadGroup SecurityManager.getThreadGroup()}.
+   *         If there is not a security manager or {@code
+   *         SecurityManager.getThreadGroup()} returns {@code null}, the group
+   *         is set to the current thread's thread group.
+   *
+   * @param  target
+   *         the object whose {@code run} method is invoked when this thread
+   *         is started. If {@code null}, this thread's run method is invoked.
+   *
+   * @param  name
+   *         the name of the new thread
+   *
+   * @param  stackSize
+   *         the desired stack size for the new thread, or zero to indicate
+   *         that this parameter is to be ignored.
+   *
+   * @throws  SecurityException
+   *          if the current thread cannot create a thread in the specified
+   *          thread group
+   *
+   * @since 1.4
+   */
+  public Thread(ThreadGroup group, Runnable target, String name, long stackSize)
+  {
+    init(group, target, name, stackSize);
+  }
+
+  /**
+   * Causes this thread to begin execution; the Java Virtual Machine
+   * calls the <code>run</code> method of this thread.
+   * <p>
+   * The result is that two threads are running concurrently: the
+   * current thread (which returns from the call to the
+   * <code>start</code> method) and the other thread (which executes its
+   * <code>run</code> method).
+   * <p>
+   * It is never legal to start a thread more than once.
+   * In particular, a thread may not be restarted once it has completed
+   * execution.
+   *
+   * @exception  IllegalThreadStateException  if the thread was already
+   *               started.
+   * @see        #run()
+   * @see        #stop()
+   */
+  // used to be synchronized. We defer this to CProver.startThread
+  public void start()
+  {
+    CProver.startThread(333);
```

Comment:
The comment that I requested here is still not here.

Reply:
Thanks for adding some more information but the information that I would like to see at this point (documenting a rather obscure design choice) is not yet in the comment.

Please add the phrases that I provided above.

Reply:
done

Reply:
Done, thanks!